tests/test_plugin_rendering.py

```diff
@@ -0,0 +1,61 @@
+from cms.api import add_plugin
```

Comment:
You've done a good job with this, I thought that your issue was a field issue in GDPP, you;re right that this is the issue and the tests that you have prove this well!

Reply:
Thanks